homeassistant/components/coolmaster/__init__.py

```diff
@@ -1 +1,22 @@
-"""The coolmaster component."""
+"""The Coolmaster integration."""
+
+
+async def async_setup(hass, config):
+    """Set up Coolmaster components."""
+    return True
+
+
+async def async_setup_entry(hass, entry):
+    """Set up Coolmaster from a config entry."""
+    hass.async_add_job(hass.config_entries.async_forward_entry_setup(entry, "climate"))
```

Comment:
`hass.async_add_job` is legacy for general use. Please use `hass.async_create_task` instead to schedule a coroutine task on the event loop.